sampleDriver/src/EvsEnumerator.cpp

```diff
@@ -164,15 +165,18 @@ bool EvsEnumerator::checkPermission() {
 
 bool EvsEnumerator::addCaptureDevice(const std::string& deviceName) {
     if (!qualifyCaptureDevice(deviceName.data())) {
-        LOG(DEBUG) << deviceName << " is not qualified for this EVS HAL implementation";
-        return false;
+        LOG(ERROR) << deviceName << " is not qualified for this EVS HAL implementation";
+        //return false;
```

Comment:
Do we need error print here ? And if return commented, why do we need this check here ?

Reply:
It is used to check if device is presented in /dev folder.
For carla streaming we need to comment this